riptide-failsafe/README.md

```diff
@@ -0,0 +1,103 @@
+# Riptide: Failsafe
+
+[![Valves](../docs/valves.jpg)](https://pixabay.com/en/wheel-valve-heating-line-turn-2137043/)
+
+[![Build Status](https://img.shields.io/travis/zalando/riptide/master.svg)](https://travis-ci.org/zalando/riptide)
+[![Coverage Status](https://img.shields.io/coveralls/zalando/riptide/master.svg)](https://coveralls.io/r/zalando/riptide)
+[![Javadoc](https://javadoc-emblem.rhcloud.com/doc/org.zalando/riptide-failsafe/badge.svg)](http://www.javadoc.io/doc/org.zalando/riptide-failsafe)
+[![Release](https://img.shields.io/github/release/zalando/riptide.svg)](https://github.com/zalando/riptide/releases)
+[![Maven Central](https://img.shields.io/maven-central/v/org.zalando/riptide-failsafe.svg)](https://maven-badges.herokuapp.com/maven-central/org.zalando/riptide-failsafe)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/zalando/riptide/master/LICENSE)
+
+*Riptide: Failsafe* adds [Failsafe](https://github.com/jhalterman/failsafe) support to *Riptide*. It offers retries
+and a circuit breaker to every remote call.
+
+## Example
+
+```java
+Http.builder()
+    .plugin(new FailsafePlugin(scheduler)
+            .withRetryPolicy(retryPolicy)
+            .withCircuitBreaker(circuitBreaker))
```

Comment:
Same here fallback?

Reply:
It wouldn't make a whole lot of sense since a single `Http` instance is used to access a remote API which usually offers more than one endpoint. Finding a fallback that would work for multiple endpoints is at best unlikely and at worst risky.

Fallbacks make more sense on an individual call-by-call basis. But I'd argue that there is something better in place for fallbacks:

```java
http.get("/")
        // dispatch/capture/...
        .exceptionally(e -> return fallback);
```

CompletableFuture already supports fallbacks, so there is no need for a library there.